### ISSUE TITLE
Make backups or custom data backups with symbols embebed in the file

### ISSUE THREAD

**Issue by arasaac-dga** [asterics/AsTeRICS-Grid#186]
This is an option commented in the past, but we annotate here in order don't forget it. It could be neccesary (especially in "Save custom data to file") to add an option to include in the grd file all the symbols/images in order to import them easily in offline devices without need to connect them to the Internet.

**Comment by klues** [comment]
For me the question arises: if a backup including image data is imported, should the image data always be imported (or only if the user isn't online)? If yes, we maybe run into the same problem again, that these backups are imported for online users and lead to much duplicated data saved on our server. However since it's a special option that's not enabled by default, maybe that's not a problem since it won't happen often?! What do you think?

**Comment by arasaac-dga** [comment]
Yes, we understand the problem you comment about and we are totally concerned about it. So, we think that a good approach it's the one you propose or import only if the user is not online. In this line, we understand that it would be possible to restore for offline users too (because they do not synchronize the information with your server). We can't find a better solution. Of course, it must be an optional feature unchecked by default. Perhaps if someone checks this option a message advising that only can be restored in offline users or users not connected can help too to be a more transparent option. Anyway, if you don't see enough clear this solution, discard at the moment.

**Comment by willwade** [comment]
Can you export to OBZ? I havent tested this so being lazy but shouldnt that too also have the image data in it?

**Comment by klues** [comment]
Yes, OBZ export is possible:

<img width="1920" height="945" alt="Image" src="https://github.com/user-attachments/assets/7ef6047b-5801-4616-8f35-99b17d4651ca" />

> but shouldnt that too also have the image data in it

Currently it takes what is in the original grid/data. If it's a link to some API picto, it uses the link, if it contains some actual image data, it uses this data.
